File: docs/user/en/query/filtering/examples/comparable/attribute-is-null.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeIs("catalogNumber", NULL)
				),
				require(
					entityFetch(
						attributeContent("code", "catalogNumber")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/comparable/attribute-is-null.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeIs("catalogNumber", Null)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "catalogNumber")
        		)
        	)
        )
	)
);